metagraph/integration_tests/test_api.py

```diff
@@ -248,39 +247,46 @@ def setUpClass(cls):
         cls.graph_client = MultiGraphClient()
         cls.graph_client.add_graph(cls.host, cls.port, cls.graph_name)
 
+        cls.sample_query_expected_rows = 98 if cls.mode == 'basic' else 99
```

Comment:
Add a comment describing why this value depends on the case. Shouldn't the number of lines of output equal the number of lines in the query (and thus not depend on the graph mode)?

Reply:
the canonical graph has more k-mers, so it finds more k-mer matches